okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/OkHttpClients.java

```diff
@@ -339,14 +334,11 @@ private static UserAgent augmentUserAgent(UserAgent agent, Class<?> serviceClass
         return augmentedAgent;
     }
 
-    private static ImmutableList<ConnectionSpec> createConnectionSpecs(boolean enableGcmCipherSuites) {
+    private static ImmutableList<ConnectionSpec> createConnectionSpecs() {
```

Comment:
What are the blockers for removing okhttp support entirely?

Reply:
I think a lot of folks still use it in tests, I’ll do a more thorough investigation later though